src/ci/docker/host-x86_64/x86_64-gnu-integration/Dockerfile

```diff
@@ -51,6 +51,9 @@ RUN sh /scripts/sccache.sh
 ENV RUST_INSTALL_DIR /checkout/obj/install
 RUN mkdir -p $RUST_INSTALL_DIR/etc
 
+# Fuchsia only supports LLVM.
+ENV CODEGEN_BACKENDS llvm
```

Comment:
Is cg_clif actually broken on Fuchsia or is it that you don't officially support it?

Reply:
As far as I know we've never tried building with it. Our build system doesn't support it currently, it has a lot of flags that assume LLVM that would need to be dealt with, and for the purpose of this builder I'm trying to shave off some time.

Reply:
It is one of those things I would _like_ to work one day, though :)

Reply:
Makes sense